nixos/modules/services/backup/sanoid.nix

```
@@ -11,6 +11,21 @@ let
     };
 
   commonOptions = {
+    frequent_period = mkOption {
+      description = ''
+        Period in minutes for frequent snapshots,
+        <varname>services.sanoid.interval</varname> should be less than or equal to this
+      '';
+      type = with types; nullOr ints.unsigned;
```

```suggestion
      type = with types; nullOr (ints.between 1 59);
```
The documentation states that it doesn't make sense to use values larger than 59.